credentials/apps/edx_credentials_extensions/edly_credentials_app/api/serializers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Serializers for edly_api
+"""
+from django.contrib.sites.models import Site
+
+from rest_framework import serializers
+
+from credentials.apps.credentials.models import ProgramCertificate, Signatory
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.fields import Base64ImageField
+
+
+class SignatorySerializer(serializers.ModelSerializer):
+    """
+    Serializer for Signatories
```

Comment:
Model is `Signatory` not `Signatories`. Kindly rename.

Reply:
Done